requests/census_household_gb_eng.json

```diff
@@ -1282,6 +1282,35 @@
         {
             "method": "GET",
             "url": "/submitted/thank-you/"
+        },
+        {
+            "method": "POST",
+            "url": "/submitted/thank-you/",
+            "data": {
+                "email": "test@email.com"
```

Comment:
We prob need to use a specific email address for Gov Notify when used for test purposes?

Reply:
That's an interesting point, should test be hitting Notify? I am not sure, but then we have a problem with how we manage the cloud function. Easiest thing to do might be to change the email to a Notify one and decide later. It also opens up the question how we handle confirmation emails locally now. Realistically we don't want to hand the key around

Reply:
There might been some value in having the smoke test email for Notify, even if we do decide not to use it in the daily benchmark.